www-client/elinks/elinks-9999.ebuild

```diff
@@ -59,53 +57,55 @@ src_prepare() {
 }
 
 src_configure() {
-	local myconf=""
+	local myconf=()
 
 	if use debug ; then
-		myconf="--enable-debug"
+		myconf+=( --enable-debug )
 	else
-		myconf="--enable-fastmem"
+		myconf+=( --enable-fastmem )
 	fi
 
 	# NOTE about GNUTSL SSL support (from the README -- 25/12/2002)
 	# As GNUTLS is not yet 100% stable and its support in ELinks is not so well
 	# tested yet, it's recommended for users to give a strong preference to OpenSSL whenever possible.
 	if use ssl ; then
-		myconf="${myconf} --with-openssl=${EPREFIX}/usr"
+		myconf+=( --with-openssl="${EPREFIX}"/usr )
 	else
-		myconf="${myconf} --without-openssl --without-gnutls"
+		myconf+=( --without-openssl --without-gnutls )
 	fi
 
-	econf \
-		--sysconfdir="${EPREFIX}"/etc/elinks \
-		--enable-leds \
-		--enable-88-colors \
-		--enable-256-colors \
-		--enable-true-color \
-		--enable-html-highlight \
-		$(use_with gpm) \
-		$(use_with brotli) \
-		$(use_with bzip2 bzlib) \
-		$(use_with guile) \
-		$(use_with idn) \
-		$(use_with javascript spidermonkey) \
-		$(use_with lua) \
-		$(use_with perl) \
-		$(use_with ruby) \
-		$(use_with tre) \
-		$(use_with X x) \
-		$(use_with zlib) \
-		$(use_enable bittorrent) \
-		$(use_enable finger) \
-		$(use_enable ftp) \
-		$(use_enable gopher) \
-		$(use_enable ipv6) \
-		$(use_enable mouse) \
-		$(use_enable nls) \
-		$(use_enable nntp) \
-		$(use_enable samba smb) \
-		$(use_enable xml xbel) \
-		${myconf}
+	myconf+=(
```

Comment:
I was actually thinking of putting it into initial declaration to avoid appending but either way works for me.